GitHub tutorial/github_tutorial.md

```diff
@@ -70,6 +70,24 @@ Wanneer je op je eigen laptop/pc werkt aan bestanden, is "origin" de alias voor
 **upstream**
 Wanneer je op je eigen laptop/pc werkt aan bestanden, is "upstream" de alias voor de GitHub (remote) repository van VNG-Realisatie. Dit gebruik je wanneer je de lokale kopie van de repository wilt actualiseren met alle door anderen doorgevoerde wijzigingen (via een `git pull upstream master`).
 
+
+**lokale files excluden van commits** Hiervoor kan niet de .gitignore worden gebruikt omdat dit bestand ook wordt ingecheckt en dit zou er dan voor zorgen dat de GitHub Actions workflows de gegenereerde bestanden niet kan committen.
+In een git repo is er een .git/info/exclude file die hetzelfde werkt als de .gitignore. Alleen wordt deze niet gecommit waardoor het alleen voor de lokale repo geldt. De volgende regels moeten worden toegevoegd om de gegenereerde bestanden te excluden voor commit.
```

Comment:
Enkele vraagjes:
* Je geeft aan 'In een git repo is er een .git/info/exclude file'. Ik neem aan dat deze er niet standaard is (er is immers ook niet standaard altijd een '.git' folder) maar dat je hem aan kunt maken. Ik zou dan ook schrijven:

> In een lokale git repo kun je een .git/info/exclude file creëren die hetzelfde werkt als de .gitignore.

Reply:
Aangepast